h2hTester.js

```diff
@@ -1,27 +1,26 @@
-const fs = require('fs')
-const dotenv = require('dotenv')
 const Client = require('ssh2-sftp-client');
-
-// open sftpGo server
-
-dotenv.config()
+const fs = require('fs')
 
 const sftp = new Client();
 
-const passphrase = process.env.PASSPHRASE
-const homePath = process.env.HOME_PATH
-
-fileToGet = Math.floor(Math.random() * 2 + 1)
+const sftpTestConnection = (configObject) => {
+  let connection
```

Comment:
A better naming would be isConnected. Usually connection var is the connection itself. But anyway see my next comment it's not really that needed anyway